pkg/addons/applier.go

```diff
@@ -81,9 +93,16 @@ func (a *Applier) Outro(ctx context.Context, k0sCfg *k0sv1beta1.ClusterConfig, e
 	if err := spinForInstallation(ctx, kcli); err != nil {
 		return err
 	}
+
+	err = createHostSupportBundle()
```

Comment:
this will only work for installs and not upgrades to this version. you will have to add something to the operator reconciliation loop for upgrades.

Reply:
Thank you. I have the createHostSupportBundle into reset and upgrade.

Reply:
The failed test is 
```
Waiting for nodes to be ready (4/5)
        Timed out waiting for 5 nodes to be ready
```
it should not be related to my PR. I tried few times. It still fail.